src/App.tsx

```diff
@@ -26,70 +28,146 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  visibleGoods.sort((goodA, goodB) => {
```

Comment:
Better to give meaningful names for the variables, even inside the array methods callbacks. Not critical in that case, but I recommend you to consider it in the future
```suggestion
  visibleGoods.sort((currentGood, nextGood) => {
```